packages/shared/src/components/squads/SquadMemberShortList.tsx

```diff
@@ -37,8 +41,9 @@ function SquadMemberShortList({
       <button
```

Comment:
I guess ideally this would just become a actual Button right?
What makes it different now?

Reply:
its used in different places so I don't know, trying to just make it work for this change

Reply:
Main issue is that internal `ProfilePicture` sizes do not match sizes of Button so I would have to introduce multiple props.

Reply:
Ah I see :( 
It's fine for now I guess, but probably something we want to fix as guidelined after this.

Reply:
For sure